src/config/configManager_test.go

```diff
@@ -49,7 +49,7 @@ feed-consumer:
       buffer: 50
 
 database:
-  driver: mysql
+  driver: sqlite3
```

Comment:
@ErenYeager713
Kindly revert the change back to mysql. Once after validation we can make this change on need basis.
Also we need to decide on the persistent directory that we will be mounting the DB to in case of K8S.